sdk/cosmosdb/Azure.ResourceManager.CosmosDB/CHANGELOG.md

```diff
@@ -1,21 +1,29 @@
 # Release History
 
+## Upcoming Release
+
+- Added support for Priority Based Execution.
+- Added support for cross account container copy.
+- Added new property CustomerManagedKeyStatus to account.
+
 ## 1.4.0-beta.4 (Unreleased)
```

Comment:
Please special the expected release date by replacing `Unreleased`, the format is `yyyy-mm-dd`.

Reply:
Added 2023-10-30 as a placeholder. Might change this after discussing with RP team